x-pack/plugins/threat_intelligence/public/components/empty_page/empty_page.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { VFC } from 'react';
+
+import { EuiEmptyPrompt, EuiImage, EuiButton, EuiLink } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import illustration from './integrations_light.svg';
+
+export const DOCS_LINK_TEST_ID = 'tiEmptyPageDocsLink';
+export const EMPTY_PROMPT_TEST_ID = 'tiEmptyPage';
+export const INTEGRATION_LINK_ID = 'tiEmptyPageIntegrationsPageLink';
+
+interface EmptyPageProps {
+  integrationsPageLink: string;
+}
+
+export const EmptyPage: VFC<EmptyPageProps> = ({ integrationsPageLink }) => {
+  const docsLink =
+    'https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html';
```

Comment:
@gchaps I confirmed with @PhilippeOberti that this link is pointing to the correct Security docs.

Reply:
Rather than having a URL in the code like this (which will not be up-to-date as our content moves), this plugin should use https://github.com/elastic/kibana/tree/main/packages/kbn-doc-links. If you need help, let me know!

Reply:
@PhilippeOberti when you set up the doc link (as per @lcawl 's suggestion in the previous comment) you can use the `threatIntelInt` keyword to connect to [these threat intel docs](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html).

Reply:
I updated the empty_page component to read from the `kbn-doc-links` plugin (though the kibana services). I made some small changes to the actual `kbn-doc-links` package (small cleanup and also added the `threatIntelInt` type that was missing in the `types.ts` file. Hopefully that is ok 😄

Reply:
@lcawl, @nastasha-solomon can you review the changes @PhilippeOberti did in kbn-doc-links, the review from @elastic/kibana-docs is required now to be able to merge this ticket